ports/stm32/boards/NUCLEO_WB55/rfcore_makefirmware.py

```diff
@@ -29,9 +29,12 @@
 # an update. See more information (and the corresponding de-obfuscation) in
 # rfcore_firmware.py as well as instructions on how to use.
 
+from imp import release_lock
```

Comment:
Unused?

Reply:
Oh, I think vscode might have added that automatically, shouldn't be there.